### PR TITLE
updated error messages for NU3018 and NU3008 docs

### PR DESCRIPTION
**Pull request by kartheekp-ms**
Fixes: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2346
Fixes: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2345

Updated error messages for `NU3018` and `NU3008` error codes as per [spec](https://github.com/NuGet/Home/blob/dev-kartheekp-ms-verify-verbosity/designs/Verify-Verbosity.md#solution).

### CODE REVIEW

File: docs/reference/errors-and-warnings/NU3008.md [hunk 1]
```diff
@@ -12,17 +12,14 @@ f1_keywords:
 
 # NuGet Error NU3008
 
-<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': The package integrity check failed.</pre>
+<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': The package integrity check failed. The package has changed since it was signed. Try clearing the local cache and run nuget operation again.</pre>
```
Comment (nkolev92) [inline docs/reference/errors-and-warnings/NU3008.md:15]: Is this actually implemented? https://github.com/NuGet/NuGet.Client/search?q=Try+clearing+the+local+cache+and+run+nuget+operation+again

What does the cache refer to here? there's some ambiguity in using cache only.

Comment (kartheekp-ms) [reply docs/reference/errors-and-warnings/NU3008.md:15]: > Is this actually implemented? 

This change is in review https://github.com/NuGet/NuGet.Client/pull/3992

Comment (kartheekp-ms) [reply docs/reference/errors-and-warnings/NU3008.md]: Updated the text in the doc as per the discussion https://github.com/NuGet/NuGet.Client/pull/3992#discussion_r611970910

**Review by JonDouglas — APPROVED** [review]
This looks good from my perspective. I assume the text is just copied from the product message merged earlier?

**Comment by kartheekp-ms** [comment]
> This looks good from my perspective. I assume the text is just copied from the product message merged earlier?

Yes Jon that is correct, text is copied from the product code.

**Pull request merged**